src/Compilers/VisualBasic/Portable/Language/LanguageFeature.vb

```diff
@@ -36,7 +36,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         InferredTupleNames
     End Enum
 
+    <HideModuleName>
     Friend Module FeatureExtensions
+
+        <Extension>
+        Friend Function IsPrototype(feature As Feature) As Boolean
```

Comment:
This method is for feature that are still in development eg `IOperation` 
Should I make it `Private` ?